recipes/tomocupy/meta.yaml

```
@@ -0,0 +1,61 @@
+{% set version = "1.0" %}
+
+package:
+  name: tomocupy
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomography/tomocupy.git
+  git_rev: develop 
+
+build:
+  # skip: True  # [cuda_compiler_version != "11.0"]
+  # skip: True  # [py != 310]
+  # skips above this line are for debugging purposes only
+  skip: True  # [cuda_compiler_version in (undefined, "None")]
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv
  script_env:
    - CUDAARCHS='all'
```
You'll want to build for all CUDA architectures instead of hardcoding "60;72;75;80". Hopefully, CMake can autodetect which archs are compatible with each CUDA version.